artiq/frontend/artiq_ddb_template.py

```diff
@@ -231,14 +236,15 @@ def process_urukul(self, rtio_offset, peripheral):
                     "sync_device": {sync_device},
                     "io_update_device": "ttl_{name}_io_update",
                     "refclk": {refclk},
-                    "clk_sel": {clk_sel}
+                    "clk_sel": {clk_sel},
+                    "clk_div" : {clk_div}
```

Comment:
just wanted to note that the space before the `:` causes a flake8 error @MorganTL